api/ShadowRoot.json

```
@@ -711,6 +711,41 @@
           }
         }
       },
+      "referenceTarget": {
+        "__compat": {
+          "mdn_url": "",
+          "spec_url": "",
+          "tags": [
+            "web-features:shadow-dom"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "133"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror",
+            "webview_ios": "mirror"
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
```

BCD considers features non-standard until they land in the spec:

```suggestion
            "standard_track": false,
```